ui/app/AppLayouts/Chat/views/communities/CommunityNewPermissionView.qml

```diff
@@ -71,7 +74,7 @@ StatusScrollView {
     // roles: itemId, text, emoji, color
     property var channelsModel: ListModel {}
 
-    property bool isPrivate
+    property bool isPrivate: false
```

Comment:
`false` by default

Reply:
Yeah, I know, just for verbosity/readability...